src/main/application/search/query-profiles/default.xml

```diff
@@ -2,4 +2,5 @@
 
 <query-profile id="default">
  <field name="presentation.timing">true</field>
+ <field name="model.language">true</field>
```

Comment:
? https://docs.vespa.ai/en/reference/query-api-reference.html#model.language - should be a language string?

Reply:
Thanks!